utils/husky-init.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const fs = require('fs');
+const shell = require('./shelljs');
+const packagePath = require('../utils/package-path')
+
+// This should be in each package's `prepare` script but we already use it to run `sf-install`.
+shell.exec('yarn husky install')
+
+function initializeHusky() {
+  try {
+    const localGitHooks = fs.readdirSync(`${packagePath}/.husky`).filter(hook => hook != '_')
+
+    if (localGitHooks.length === 0) {
+      shell.exec("yarn husky add .husky/commit-msg 'yarn commitlint --edit'")
+      shell.exec("yarn husky add .husky/pre-commit 'yarn lint && yarn pretty-quick --staged'")
+      shell.exec("yarn husky add .husky/pre-push 'yarn build && yarn test'")
+    }
+  } catch (err) {
+    if (err.code === "ENOENT") {
+      const errorHeader = chalk.red('ERROR: ');
+      const errorMsg = ".husky folder wasn't found, try running `yarn husky install` to finish the install";
```

Comment:
This is just in case `fs.readdirSync` doesn't find the `.husky` folder. Ideally this will never happen because `husky install` should create it but if that happens we suggest the user to run to try installing husky again manually.